src/Data/Textile/Material.elm

```diff
@@ -2,22 +2,35 @@ module Data.Textile.Material exposing
     ( CFFData
     , Id(..)
     , Material
+    , Spinning(..)
     , decodeList
+    , decodeSpinning
     , encode
     , encodeId
+    , encodeSpinning
     , findById
+    , getAvailableSpinningProcesses
+    , getDefaultSpinning
     , getRecyclingData
+    , getSpinningElec
     , groupAll
     , idToString
+    , spinningFromString
+    , spinningToLabel
+    , spinningToString
+    , wasteForSpinning
```

Comment:
Nit, just an observation: from a distance, it almost looks like `Spinning` could be a module.

Reply:
Yeah, I thought about that, and didn't... but I'm doing it right now, you're right, it makes more sense 👍